tests/test_density_RDF.py

```diff
@@ -15,11 +15,8 @@ def test_generateR(self):
             nbins = int((r_max - r_min) / dr)
 
             # make sure the radius for each bin is generated correctly
-            r_list = np.zeros(nbins, dtype=np.float32)
-            for i in range(nbins):
-                r1 = i * dr + r_min
-                r2 = r1 + dr
-                r_list[i] = 2.0/3.0 * (r2**3.0 - r1**3.0) / (r2**2.0 - r1**2.0)
+            r_list = np.array([r_min + dr*(i+1/2) for i in range(nbins) if
+                               r_min + dr*(i+1/2) < r_max])
```

Comment:
I'm not convinced this is accurate. I would have expected `np.arange(r_min, r_max, dr) + dr/2`.

It looks like you might include a final bin if the bin _center_ is less than `r_max`, but I think partial bins are never included. This test becomes less tricky if we move to `nbins` instead of `dr` in the constructor.

Reply:
I think what you're missing is that `nbins` prevents you from making more bins than what you should. I agree that we can simplify things if we move to a number of bins, but that's for a future PR. My math should be equivalent to what you would expect from `np.arange`; that function's docs explicitly suggest not to use it when the step is non-integral, so I took them at their word, but I do think that practically speaking these approaches are equivalent.